### ISSUE TITLE
MISP not fetching feeds with public IP on instance

### ISSUE THREAD

**Issue by ojaschauhan** [MISP/MISP#7631]
## Please consider the following notes
- Critical security bugs can be reported, preferably PGP encrypted, by mail to: info@circl.lu
- Bug reports and feature requests can be filed as an issue on github: https://github.com/MISP/MISP/issues
- For interactive support please join the Gitter chat on: https://gitter.im/MISP/MISP

The official documentation of MISP can be found here: https://www.circl.lu/doc/misp/
We also periodically do user/admin/developer trainings and have our training material online: https://www.circl.lu/services/misp-training-materials/

Nevertheless you can of course file a Support request as an issue. Please be as precise as possible and fill the template as detailed as possible too.
Please remove this text until the line below. Thanks a lot.
--------8<------

### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Support
| OS version (server)       | Debian, Ubuntu, CentOS, RedHat, ...
| OS version (client)       | MacOS, Win10, Ubuntu, ...
| PHP version               | 5.4, 5.5, 5.6, 7.0, 7.1... (if relevant)
| MISP version / git hash   | 2.4.XX, hash of the commit
| Browser                   | If applicable

### Support Questions

I am trying to setup misp under SSO from okta and no access by IP. But as I setup instance without IP it shows connection error on fetch feeds. But with a Public IP its all good. Anything I am missing here?

**Comment by ojaschauhan** [comment]
<img width="1363" alt="Screenshot 2021-08-09 at 1 53 03 PM" src="https://user-images.githubusercontent.com/88473482/128678006-36f1b3d1-0641-4edb-96ee-ace7b8100464.png">


Could not fetch manifest for feed: Fetching the '<url>' failed with exception: Connection timed out

**Comment by righel** [comment]
Hello @ojaschauhan,
What do you mean "without IP" using a dns hostname?
Could you check if you can reach the target instance network?

**Comment by ojaschauhan** [comment]
@righel  yes I checked and I am able to reach the network. I even ran a reachability analyser from my vpc to instance and it was a success.
What I mean is, as I remove my eip from instance i cannot access the tool.

**Comment by righel** [comment]
Hello @ojaschauhan,
I don't fully understand your issue or the steps you are following, would you care to elaborate a bit more on the exact steps to reproduce your issue?
